File: 图片分享/springboot（后端）/share/src/main/java/com/img/share/pojo/Img.java
package com.img.share.pojo;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Img {
    private Integer iid;
    private String iname;
    private String isrc;
    private String uploaddate;
    private User user;
    private Integer pageview;
    private List<Comment> comment;
    private List<User> likUsers;
}

File: 图片分享/vue3-web（前端）/imgshare/src/pojo/Img.ts
import type { Commentt } from "./Commentt"
import { User } from "./User"

export class Img {
    iid: number = -1
    iname: string = ''
    isrc: string = ''
    uploaddate: string = ''
    user: User = new User()
    pageview: number = 0
    comment: Commentt[] = []
    likeUsers: User[] = []
    constructor(
    ) {

    }
}